torchbox/torchbox.com
# Serverside implementation of mode switcher

[Link to Ticket]()

### Description of Changes Made

Server-side fallback to set the mode cookie if JavaScript is disabled.
The documentation will be updated in a separate pull request when all the light mode work is finished.

### How to Test

- Delete any existing `mode` or `torchbox-mode` cookies stored in your browser
- With dev tools open, disable javascript by typing `option shift p` then at the `run` prompt` start to type 'JavaScript` - click on the option to disable it
- Load any page on your local build
- Check that there isn't a `torchbox-mode` cookie set
- Switch to light mode using the unstyled form
- The page should reload
- Check that a `torchbox-mode` cookie is now set with the value `light`
- Check that the light style persists as you browse the site
- Repeat the test switching back to dark mode

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
